x-pack/plugins/fleet/public/components/enrollment_instructions/manual/index.tsx

```diff
@@ -23,11 +23,46 @@ function getfleetServerHostsEnrollArgs(
   return `--url=${fleetServerHost || `FLEET_SERVER_HOST`} --enrollment-token=${apiKey}${proxyArgs}`;
 }
 
+export const getDownloadBaseUrl = (downloadSource?: DownloadSource) => {
+  return downloadSource?.host.endsWith('/')
+    ? downloadSource.host.substring(0, downloadSource.host.length - 1)
+    : 'https://artifacts.elastic.co/downloads';
```

Comment:
Nit: you could use the constant in https://github.com/elastic/kibana/blob/a2e995a9e735a50f54a314197ab1c2812a8416fb/x-pack/plugins/fleet/common/constants/download_source.ts#L9

Reply:
good call, thank you!